.github/workflows/build.yml

```diff
@@ -43,6 +43,29 @@ jobs:
         name: linux
         path: ../linux.tar.gz
 
+  linux-static:
+    runs-on: ubuntu-20.04
+    if: github.event_name == 'push'
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - name: build
+      run: cd src && ./make.bash
+      env:
+        CGO_ENABLED: "0"
```

Comment:
I think just doing this will be too problematic on callers (casual developers in the tree ... we can't just force it back to true easily. We want to have the same magic behavior as not setting)

So I think we want to modify our Go tree as well.

But you can start with this and see.

Reply:
I want this right now so I can try to update all our toolchain goop to use static builds and see if I can make it all work out. If it does, we can tweak the default behavior before rolling it out more, yeah.